lib/iris/analysis/__init__.py

```diff
@@ -1263,13 +1263,25 @@ def _percentile(data, percent, fast_percentile_method=False, **kwargs):
     If a new additive dimension is formed, then it will always be the last
     dimension of the resulting percentile data payload.
 
+    Args:
+
+    * data (array-like)
+        array from which percentiles are to be calculated
+    * axis (int or tuple of int)
+        Axis or axes along which the percentiles are computed
+        (handled by decorator)
```

Comment:
It feels weird to describe an argument in the docstring that doesn't feature in the signature. Is this the normal way of doing that?

Reply:
Mmmm not sure, and I dithered over it before I did it!  Happy to take it out.